src/sass/login.scss

```diff
@@ -1,5 +1,190 @@
 @import "shared-variables";
 
+.login {
+  background: $color-white;
+
+  hr {
+    border-color: $color-gray-lighter;
+  }
+
+  @include media-maxwidth($small-screen) {
+    .hide-for-small {
+      display: none;
+    }
+
+    .explanation-content {
+      padding: 1em 2em;
+    }
+
+    .help-info {
+      padding: 2em !important;
+    }
+  }
+
+  @include media($small-screen) {
+    .hide-for-medium-up {
+      display: none;
+    }
+  }
+
+  .mobile-explanation {
+    padding: 1em 0;
+
+    p {
+      margin-top: 0;
+    }
+  }
+
+  .container {
+    padding: 2em 0;
+  }
+
+  .logo {
+    text-align: center;
+    border-bottom: solid 2px $color-gray-light-alt;
+    padding: 1em 0;
+
+    a {
+      &:hover {
+        background-color: transparent;
+      }
+      display: inline-block;
+    }
+  }
+
+  .fed-warning {
+    font-size: 0.9em;
+    margin-bottom: 2em;
+
+    p {
+      margin: 0.5em 0;
+    }
+  }
+
+  .help-info {
+    padding: 2em 0;
+
+    p:first-of-type {
+      margin-top: 0;
+    }
+  }
+
+  .signin-actions-container {
+    text-align: center;
+    margin-right: 2em;
+
+    @include media-maxwidth($small-screen) {
+      margin-right: 0;
+    }
+
+    div:first-child {
+      background: $color-gray-light-alt;
+    }
+
+    border: solid 2px $color-gray-light-alt;
+
+    .top-banner {
+      text-align: center;
+      display: flex;
+      align-items: center;
+      padding: 0.25em 0;
+      font-style: italic;
+
+      div:first-child {
+        margin: 0 auto;
+      }
+
+      img {
+        height: inherit;
+        vertical-align: baseline;
+        padding: 0 0.25em;
+      }
+    }
+
+    .signin-actions {
+      padding: 2em;
```

Comment:
@bshyong the intention was for the larger side padding to only apply to large screens. Is it possible to add an include here so that large screen will be `2em 2em 1em`, and small will be `2em 1em 1em`?